RadarSDK/RadarUserDefaults.m

```diff
@@ -0,0 +1,175 @@
+//
+//  RadarUserDefaults.m
+//  RadarSDK
+//
+//  Created by Kenny Hu on 12/14/23.
+//  Copyright © 2023 Radar Labs, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "RadarUserDefaults.h"
+#import "RadarFileStorage.h"
+
+@implementation RadarUserDefaults 
+
+
+static NSString *const kCompletedMigration = @"radar-completed-migration";
+
+
++ (instancetype)sharedInstance {
+    static dispatch_once_t once;
+    static id sharedInstance;
+    dispatch_once(&once, ^{
+        sharedInstance = [self new];
+    });
+    return sharedInstance;
+}
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        NSString *documentsDirectory = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES).firstObject;
+        self.settingsFileDir = [documentsDirectory stringByAppendingPathComponent:@"radar_settings"];
```

Comment:
We're storing "replays" in here right? I don't know if "radar_settings" is correct.